interface/weno/import_pharmacies.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
For the above interface/weno/img/progress.gif file, better to place this at public/images/